app/views/errors/internal_server_error.html.erb

```diff
@@ -0,0 +1,5 @@
+<div>
+  <h1 class="text-6xl mb-2">500</h1>
+  <h2><%= t(".title") %></h2>
+  <p><%= t(".subtitle") %></p>
```

Comment:
This is a great idea!
I struggled a bit with this text because it also lacks information about who to contact.
Eventually I decided that adding that was not part of the scope for the task.

I will create a new task in the muffi template which is about adding a contact and debug information to the 500 page.

Reply:
https://app.asana.com/0/1142794766483633/1207043560713654